development/webpack/webpack.config.ts

```diff
@@ -159,15 +160,15 @@ const plugins: WebpackPluginInstance[] = [
     // Make a global `Buffer` variable that points to the `buffer` package.
     Buffer: ['buffer', 'Buffer'],
     // Make a global `process` variable that points to the `process` package.
-    process: 'process/browser',
+    process: 'process/browser.js',
```

Comment:
I removed `fullySpecified: false` below, because the module that was previously causing the issue has been fixed. But this caused these two polyfill's to become a new problem, because they weren't `fullySpecified` themselves. Now they are :-)